notes/lattice_cryptography.tex

```diff
@@ -128,28 +127,29 @@ \subsection{Introduction to Lattices}
     \label{fig:SVP for Gamma}
 \end{figure}
 
+\noindent However, critically, the above hardness classifications are believed to apply to both classical computers and quantum computers. Thus to build a quantum-resistant cryptosystem, the strategy is to find constructions whose security can reduce to the $\gapsvpy$ problem.
```

Comment:
should mention that the best known algorithm complexities for \gapsvpy apply to quantum computers as well